tests/testthat/test-tmb-methods.R

```diff
@@ -173,6 +173,19 @@ test_that("predict will return NA if data contains NA in covariates", {
   )
 })
 
+test_that("predict can give unconditional predictions", {
```

Comment:
thanks Liming, can we also add in addition
- one test to check that it works with formula-based response variable
- one test to see that we can modify the prediction result when changing the saved coefficients (similar as it is needed for the interface to the {marginaleffects} package)